sdk/keyvault/azure-keyvault-certificates/tests/certificates_test_case.py

```diff
@@ -15,11 +15,6 @@ def setUp(self):
 
     def tearDown(self):
         super(KeyVaultTestCase, self).tearDown()
-        if self.is_live:
```

Comment:
you don't need the tearDown at all if the only call is "super"

Reply:
makes sense, thanks so much!